docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+
+    Hack the function of get cache size:
+
+        ``unsigned int dnnl::impl::cpu::platform::get_per_core_cache_size(int level)``
+
+    to make it return emulated cache size in analyzed stage, the simplest way is to leverage environment variable to pass the emulated cache size, for example:
+
+    .. code-block:: cpp
+
+        #if defined(SELECTIVE_BUILD_ANALYZER)
+            if (level == 2) {
+                const char* L2_cache_size = std::getenv("OV_CC_L2_CACHE_SIZE");
+                if (L2_cache_size) {
+                    int size = std::atoi(L2_cache_size);
+                    if (size > 0) {
+                        return size;
+                    }
+                }
+            } else if (level == 3) {
+                const char* L3_cache_size = std::getenv("OV_CC_L3_CACHE_SIZE");
+                if (L3_cache_size) {
+                    int size = std::atoi(L3_cache_size);
+                    if (size > 0) {
+                        return size;
+                    }
+                }
+            } else if (level == 1) {
+                const char* L1_cache_size = std::getenv("OV_CC_L1_CACHE_SIZE");
+                if (L1_cache_size) {
+                    int size = std::atoi(L1_cache_size);
+                    if (size > 0) {
+                        return size;
+                    }
+                }
+            }
+        #endif
+
+- CPU core number emulation
+
+    Leverage ``numactl`` tool to control core number.
+
+    .. code-block:: sh
+
+        python thirdparty/itt_collector/runtool/sea_runtool.py --bindir ${OPENVINO_LIBRARY_DIR} -o ${MY_MODEL_RESULT} ! numactl -C 0-$core_num ./benchmark_app -niter 1 -nireq 1 -m ${MY_MODEL}.xml
+
+
+Once the SKUs are decided, we can collect cpu information(CPUID, L1/L2/L3 cache size, core number) and then profile each pair of (CPUID, L1/L2/L3 cache size, core number) to get profiling csv files, then apply all csv files to generate final conditional compilation package.
+
+Such work can be done in a single machine as below: 
+
+    .. code-block:: sh
+
+        export OV_CC_L1_CACHE_SIZE=<L1 cache size>
+        export OV_CC_L2_CACHE_SIZE=<L2 cache size>
+        export OV_CC_L3_CACHE_SIZE=<L3 cache size>
+        python thirdparty/itt_collector/runtool/sea_runtool.py --bindir ${OPENVINO_LIBRARY_DIR} -o ${MY_MODEL_RESULT} ! sde -spr -- numactl -C 0-$core_num ./benchmark_app -niter 1 -nireq 1 -m ${MY_MODEL}.xml
+
+Do above steps for each SKUs information(CPUID, L1/L2/L3 cache size, core number) to collect all generated statistics csv files together, and provide them to build resulting OpenVINO package.
+
+    .. code-block:: sh
+
+        cmake -DSELECTIVE_BUILD=ON -DSELECTIVE_BUILD_STAT=${ABSOLUTE_PATH_TO_STATISTICS_FILES}/*.csv -DENABLE_PROFILING_ITT=OFF ..
+        cmake --build .
+
+
+How to enable conditional compilation on Windows
+################################################
+
+Reference: https://github.com/openvinotoolkit/openvino/wiki/StaticLibraries
```

Comment:
Find detailed information in the `Building OpenVINO static libraries <https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/static_libaries.md>`__ document.